R/volcano_protti.R

```diff
@@ -39,12 +40,13 @@
 #' target = "Q9Y6K9",
 #' title = "Finding Nemo",
 #' x_axis_label = "log2(fold change) treated vs untreated",
+#' y_axis_label = "-log10(p-value)",
 #' foldchange_cutoff = 2,
 #' significance_cutoff = 0.05,
 #' interactive = TRUE
 #' )
 #' }
-volcano_protti <- function(data, grouping, foldchange, significance, method, protein_identifier = NULL, target = NULL, title = "Volcano plot", x_axis_label = "log2(fold change)", foldchange_cutoff = 2, significance_cutoff = 0.01, interactive = FALSE)
+volcano_protti <- function(data, grouping, foldchange, significance, method, protein_identifier = NULL, target = NULL, title = "Volcano plot", x_axis_label = "log2(fold change)", y_axis_label = "-log10(q-value)", foldchange_cutoff = 1, significance_cutoff = 0.01, interactive = FALSE)
```

Comment:
should the default not be 4 so log2 of 2. Paola always says this is what we in general take as a default. 
A default of 1 actually means no change right? log2 of 1 is 0.

Reply:
log2 of 4 is two. So the default should be two, if you say that a foldchange of 4 should be applied. I think that the default should be a foldchange of 2, so the log2(FC) should be 1. 

I will change everything to log2 now in the function. I think then these things will be less confusing.

Reply:
yes a log2 of 4 is 2. So Paola wants a log2 of 2 so a fold change of 4 as default. This is because usually the variance is too for log2 fold changes below 2 if I remember correctly. But maybe now it is different. 

Well the idea was to have the cutoff not as log2 fold change but as fold change. So in the function these fold changes need to be converted so that they fit the axis of the volcano plot which is log2 fold change.

Reply:
A fold change of 4 is extremely high. I remember hearing this number before, for metabolite data this might make sense, for PPIs I have never seen such high fold changes before. 

I know this was the idea, I think, however, that most people calculate log2FC anyway, so it easier to provide just that. I think the main problem that Viviane had was that from the variable names it was not clear what they should be (i.e. if they should be transformed or not).

Reply:
In the LiP Quant paper they used an even lower cutoff with 1.5, log2 transformed that would be 0.58

Reply:
okay that is good! that resolves the question. 
I think lip-quant is a bit special because it does not use binary information like a volcano. So maybe it is a bit better in distinguishing between real change and noise. But you are right, for metabolite it makes sense. We can use a default of log2FC 1.